### ISSUE TITLE
Copy Path Should Work for Files Outside of Current Working Directory

### ISSUE THREAD

**Issue by Haz3lnut** [warpdotdev/Warp#4636]
### Describe the solution you'd like?

When selecting filenames in a path outside of the current working directory (eg, "ls -l /etc") the "Copy Path" option does not appear in the right-click context menu. Also would like to be able to click on a filename and be given the option to open.

I almost put this in bugs, but I assume it's simply not implemented yet.

### Is your feature request related to a problem? Please describe.

Yes. It's frustrating to have to cd into another directory to be able to select files and access the full context menu.

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by dannyneira** [comment]
Thanks for this feature request @Haz3lnut as a possible workaround, double-click on the path, copy should show up regardless of your cwd.

To anyone else interested in this feature, please add a 👍 to the original post at the top to signal that you want this feature, and subscribe if you'd like to be notified.

**Comment by Haz3lnut** [comment]
Yes, there is a Copy option, but it only copies the filename, not the full path. And there is no option to open the file with the default app.

**Comment by rndjams** [comment]
While investigating the broader clickable-filename behavior I filed #9908, which I think explains the root cause of the "files outside CWD are not clickable" case you describe here. The detector resolves clickable-path candidates against the block's CWD only; bare filenames in `ls DIR/` output aren't rooted at CWD, so they're rejected. #9908 includes a proposed fix that adds the listed argument directory as a resolution root — which naturally makes files in `ls DIR/` output clickable again. Linking in case it's useful context for whoever triages either.
